diff -Naur asperite/laf/cmake/FindSkia.cmake aseprite-patched/laf/cmake/FindSkia.cmake
--- asperite/laf/cmake/FindSkia.cmake	2022-05-31 06:44:00.000000000 +0700
+++ aseprite-patched/laf/cmake/FindSkia.cmake	2022-06-13 15:47:40.498318828 +0700
@@ -122,14 +122,18 @@
   set(ZLIB_LIBRARIES ${ZLIB_LIBRARY})
 endif()
 
-set(FREETYPE_FOUND ON)
-find_library(FREETYPE_LIBRARY freetype2 PATH "${SKIA_LIBRARY_DIR}" NO_DEFAULT_PATH)
-set(FREETYPE_LIBRARIES ${FREETYPE_LIBRARY})
-set(FREETYPE_INCLUDE_DIRS "${SKIA_DIR}/third_party/externals/freetype/include")
+if(NOT USE_SHARED_FREETYPE)
+  set(FREETYPE_FOUND ON)
+  find_library(FREETYPE_LIBRARY freetype2 PATH "${SKIA_LIBRARY_DIR}" NO_DEFAULT_PATH)
+  set(FREETYPE_LIBRARIES ${FREETYPE_LIBRARY})
+  set(FREETYPE_INCLUDE_DIRS "${SKIA_DIR}/third_party/externals/freetype/include")
+endif()
 
-find_library(HARFBUZZ_LIBRARY harfbuzz PATH "${SKIA_LIBRARY_DIR}" NO_DEFAULT_PATH)
-set(HARFBUZZ_LIBRARIES ${HARFBUZZ_LIBRARY})
-set(HARFBUZZ_INCLUDE_DIRS "${SKIA_DIR}/third_party/externals/harfbuzz/src")
+if(NOT USE_SHARED_HARFBUZZ)
+  find_library(HARFBUZZ_LIBRARY harfbuzz PATH "${SKIA_LIBRARY_DIR}" NO_DEFAULT_PATH)
+  set(HARFBUZZ_LIBRARIES ${HARFBUZZ_LIBRARY})
+  set(HARFBUZZ_INCLUDE_DIRS "${SKIA_DIR}/third_party/externals/harfbuzz/src")
+endif()
 
 set(SKIA_LIBRARIES
   ${SKIA_LIBRARY}
